mm2src/coins/solana.rs

```diff
@@ -806,3 +1082,321 @@ impl MmCoin for SolanaCoin {
 
     fn on_token_deactivated(&self, _ticker: &str) {}
 }
+
+#[derive(Debug)]
+pub enum AtomicSwapInstruction {
```

Comment:
As stated here https://github.com/KomodoPlatform/satomic-swap/pull/1#discussion_r1638004840 ,  we can remove the block of code to the end of the file and import it from the satomic-swap repo.